pkg/network/network.go

```diff
@@ -97,6 +97,8 @@ func inspect(addr string) (*Parameters, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed listing network interfaces: %w", err)
 	}
+
+ifLoop:
 	for _, iface := range ifaces {
```

Comment:
for better code readability, how about we move this to a func

and instead of break, we just return

something like 
```
parseSomething(ifs []Interface ) *Parameters{}
```

Reply:
We need to change `network` as well if we find the interface (and don't touch it if not)